app/validationFunctions.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+let R = require('ramda')
+
+module.exports = {
+  isEmptyOrHasSpace (input) {
```

Comment:
Its just an es6 shorthand, sorta like get/set. And I guess it makes sense because you're already in an object scope and don't need fat arrow to bind 'this'.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions